internal/vendor/wiz.go

```diff
@@ -170,8 +191,8 @@ type CloudOrganization struct {
 
 // UpdateProjectInput struct
 type UpdateProjectInput struct {
-	ID    string             `json:"id"`
-	Patch UpdateProjectPatch `json:"patch"`
```

Comment:
Patch is part of the GraphQL schema.  Even though not in use with the changes, it should be retained.